proto/internal/temporal/server/api/adminservice/v1/request_response.proto

```diff
@@ -172,6 +172,25 @@ message GetWorkflowExecutionRawHistoryV2Response {
   repeated int64 history_node_ids = 4;
 }
 
+message GetWorkflowExecutionRawHistoryRequest {
+  reserved 1;
+  string namespace_id = 9;
+  temporal.api.common.v1.WorkflowExecution execution = 2;
+  int64 start_event_id = 3;
+  int64 start_event_version = 4;
+  int64 end_event_id = 5;
+  int64 end_event_version = 6;
+  int32 maximum_page_size = 7;
+  bytes next_page_token = 8;
```

Comment:
instead of creating a new API, can we extend GetWorkflowExecutionRawHistoryV2 with mode? default is exclusive/exclusive, but can specify with inclusive/inclusive mode?

Reply:
I think we(CGS) made an agreement that we are creating new api instead of supplying a flag.